packages/playground/src/components/pull-request/pull-request-side-bar.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import {
+  Plus,
+  CheckCircleSolid,
+  WarningTriangleSolid,
+  WarningCircleSolid,
+  ChatBubbleQuestionSolid
+} from '@harnessio/icons-noir'
+import { Else, Match, Truthy } from 'react-jsx-match'
+import { Button, Avatar, AvatarFallback } from '@harnessio/canary'
+import { EnumPullReqReviewDecision, PullReqReviewDecision, ReviewerData } from './utils'
+
+interface PullRequestSideBarProps {
+  reviewers?: ReviewerData[]
+  // repoMetadata?: any
+  //   RepoRepositoryOutput
+  processReviewDecision: (
+    review_decision: EnumPullReqReviewDecision,
+    reviewedSHA?: string,
+    sourceSHA?: string
+  ) => EnumPullReqReviewDecision | PullReqReviewDecision.outdated
+  pullRequestMetadata?: { source_sha: string }
+  //   TypesPullReq
+  refetchReviewers: () => void
+}
+
+const PullRequestSideBar = (props: PullRequestSideBarProps) => {
+  const { reviewers, pullRequestMetadata, processReviewDecision } = props
+  // add toaster error message
+  //   const { showError } = useToaster()
+
+  const getInitials = (name: string) => {
```

Comment:
this can/should be moved to a separate util.. 
actually, doesn't the Avatar component already handle this?

Reply:
this isn't something handled  in avatar component from shadcn